src/pages/success-stories/moondao.page.tsx

```diff
@@ -0,0 +1,111 @@
+import { CaseStudyContentContainer } from 'components/CaseStudies/CaseStudyContentContainer'
+import ExternalLink from 'components/ExternalLink'
+import SectionHeader from 'components/SectionHeader'
+import { AppWrapper } from 'components/common'
+import { CASE_STUDY_PROJECTS } from 'constants/successStoryProjects'
+import Image from 'next/image'
+
+export default function MoonDAOPage() {
+  const moonDAOSuccessStoryProject = CASE_STUDY_PROJECTS[1]
+
+  const fundingConfig: CaseStudiesConfigurationProps = {
+    cycleNumber: 1,
+    cycles: t`Locked`,
+    duration: t`30 days`,
+    payouts: t`Unlimited`,
+    tokenName: 'MOONEY',
+    tokenIssuanceRate: '500,000 (+ 500,000 reserved)',
+    reservedRate: 50,
+    issuanceReductionRate: 0,
+    redemptionRate: 100,
+    ownerTokenMinting: false,
+    editDeadline: t`No deadline`,
+  }
+
+  return (
+    <AppWrapper>
+      <CaseStudiesHeader
+        header="/assets/moondao-casestudy-banner.png"
+        title="MoonDAO"
+        subtitle={
+          <Trans>
+            How a community of space enthusiasts raised $8.3 million to send a
+            DAO member to space on a Blue Origin rocket
+          </Trans>
+        }
+        totalRaised={2623.7}
+        totalPayments={2677}
+        summary={
+          <Trans>
+            MoonDAO is a worldwide collective united by the mission of
+            decentralizing access to space research and exploration. With
+            privatized space exploration on the rise, MoonDAO is uniting space
+            enthusiasts from around the globe to help make it more accessible
+            while continuing to push research forward. In 2021, they launched a
+            campaign to buy two tickets on a Blue Origin rocket and raised over
+            $8 million on Juicebox. A year later, they sent Coby Cotton from
+            viral YouTube channel Dude Perfect to space as their first
+            astronaut.
+          </Trans>
+        }
+        projectUrl="/p/moondao"
+        fundingConfig={fundingConfig}
+      />
+      <CaseStudyContentContainer currentProject={moonDAOSuccessStoryProject}>
+        <p>
+          Following the launch of their token $MOONEY, the next goal on
+          MoonDAO’s roadmap was to buy two tickets to space on a Blue Origin
+          rocket. They set the project’s Payouts to Infinite in order to raise
+          as much as possible during their campaign and be able to distribute
+          all of the funds. Using a reserved rate of 50%, the MoonDAO multi-sig
+          received half of all newly minted tokens to build up a treasury of
+          both ETH and $MOONEY that could later be used for bounties and
+          compensation.
+        </p>
+
+        <Image
+          src="/assets/moondao-casestudy-banner2.png"
+          alt="MoonDAO one small step"
+          width={592}
+          height={256}
+          className="pb-5"
+        />
+        <div>
```

Comment:
Soz, i guess my comment above actually applies to this section

https://github.com/jbx-protocol/juice-interface/pull/3455/files#r1173325819